ci-jobs/templates/package.yml

```diff
@@ -7,7 +7,9 @@ parameters:
   pool: ''
   xvfb: false
   target: ''
-  buildScript: 'npx electron-builder build --publish always'
+  buildScript: |
+    npx electron-builder build -w --ia32 --x64 --publish always
```

Comment:
I don't think this will work in Mac. I don't believe Mac is capable of building a Windows application. You can test by just manually triggering this pipeline in Azure and seeing what happens.

Reply:
Ok. Will check.

I built with the -w option on my macOS for test. 
https://github.com/appium/appium-desktop/issues/1249#issuecomment-589919918 and I also tested the exe on my Windows machine, but I think it's safer to test so.

Reply:
@dpgraham 
Could you share the Azure login data with 1pass or add fly.49.89.over@hotmail.co.jp
 to the team to be able to trigger a pipeline on Azure project?

--

I've changed to override buildScript only on Windows env in be4fe7b